docs/pages/guides/config-plugins.mdx

```diff
@@ -671,7 +692,8 @@ export const withCustomConfig: ConfigPlugin<string> = (config, id) => {
 
 ### Modifying the iOS Podfile
 
-The iOS Podfile is the config file for CocoaPods, the dependency manager on iOS. Think of Podfile like package.json but for iOS. The Podfile is a ruby file (application code), which means you **cannot** safely modify it from Expo config plugins, and should opt towards another approach, like Expo Autolinking hooks (citation needed).
+The iOS Podfile is the config file for CocoaPods, the dependency manager on iOS. Think of Podfile like package.json but for iOS.
+The Podfile is a ruby file (application code), which means you **cannot** safely modify it from Expo config plugins, and should opt towards another approach, like Expo Autolinking hooks (citation needed).
```

Comment:
```suggestion
The **Podfile** is a ruby file (application code), which means you **cannot** safely modify it from Expo config plugins and should opt for another approach, such as [Expo Autolinking](modules/autolinking) hooks.
```
I think we can remove the "citation needed" part and link Autolinking docs here.